nmdc_runtime/api/db/mongo.py

```diff
@@ -74,3 +95,14 @@ def activity_collection_names(mdb: MongoDatabase) -> Set[str]:
         "functional_annotation_set",
         "genome_feature_set",
     }
+
+
```

Comment:
Add documentation for why this is required

Reply:
I assume this question is re: the addition of the `mongodump_excluded_collections()` helper. This was used to sub-select from a full database dump to ensure only needed collections for testing referential integrity code and avoiding storing personally identifiable information (PII) as obtained from the full dumps.

However, this helper more properly represents unfinished work to enable regular posting of "public" backups of nmdc metadata to streamline CI, dev, etc. If I had to do it over again, I'd have left this code out of this PR and reserved it for a separate dedicated feature branch.